### PR TITLE
update broken links in UserGuide

### PR DESCRIPTION
**Pull request by StephenGriffies**
Update broken links in UserGuide and remove link to Breakaway Labs, replacing it with link to Github.

### CODE REVIEW

File: _docs/02-user_guide.md [hunk 1]
```diff
@@ -54,18 +54,29 @@ The purpose of this section is to outline certain features of MOM.
...
+* [A Technical Guide to MOM4](/assets/pdfs/MOM4_manual.pdf) by Stephen.Griffies@noaa.gov, Matthew.Harrison@noaa.gov, Ronald.Pacanowski@noaa.gov, and Tony.Rosati@noaa.gov. This is the primary reference for MOM4p0. It contains details about some of the numerical algorithms and diagnostics. Reference to MOM4p0 in the literature should refer to this document:
 
        
         A Technical Guide to MOM4
         GFDL Ocean Group Technical Report No. 5
         S.M. Griffies, M.J. Harrison, R.C. Pacanowski, and A. Rosati
         NOAA/Geophysical Fluid Dynamics Laboratory
         August 2004
-        Available on-line at http://www.mom-ocean.org/web/docs
+        Available on-line at https://github.com/mom-ocean/mom-ocean.github.io
```
Comment (aidanheerdegen) [inline _docs/02-user_guide.md:65]:
```suggestion
        Available on-line at https://mom-ocean.github.io
```

File: _docs/02-user_guide.md [hunk 2]
```diff
@@ -74,7 +85,7 @@ In addition to this online user guide, documentation for MOM is provided by the
         NOAA/Geophysical Fluid Dynamics Laboratory
         June 2012
         620 + xiii pages 
-        Available on-line at http://www.mom-ocean.org/web/docs
+        Available on-line at https://github.com/mom-ocean/mom-ocean.github.io
```
Comment (aidanheerdegen) [inline _docs/02-user_guide.md:88]:
```suggestion
        Available on-line at https://mom-ocean.github.io
```

**Review by aidanheerdegen — CHANGES_REQUESTED** [review]
I think the other links should work, but we won't really know until it is committed, but easy enough to fix if they're broken

**Comment by aidanheerdegen** [comment]
Hi Steve,

Just checking you wanted to keep those links like so:

Available on-line at https://github.com/mom-ocean/mom-ocean.github.io

and not

Available on-line at https://mom-ocean.github.io

as suggested?

Cheers

Aidan

**Comment by StephenGriffies** [comment]
Good point.  It is more suitable to point to the shorter path.  I will make the change and do another pull request.  Thanks @aidanheerdegen

**Pull request merged**